packages/example/src/screens/HeadlessCheckoutScreen.tsx

```diff
@@ -80,6 +81,9 @@ export const HeadlessCheckoutScreen = (props: any) => {
       iOS: {
         urlScheme: 'merchant://primer.io',
       },
+      stripeOptions: {
+          publishableKey: "pk_test_51O8zfQKUK6bXIdC2xPvSS6UvriE9kwpvttwB7H9PAzhNAZoGzLgiDOyd4WooozeWHxoRrKo6b2VjZYxNCMX3W7bk00rMDivjau"
+      },
```

Comment:
Do we need to add this to CI?

Reply:
On Android and iOS the test publishable key is already in the repos, I followed the same approach here. @semirp, what do you think?

Reply:
I'd agree with @gt-prime that this could've been moved to env/config. Not sure if anyone can do anything with this test key anyways.